src/pages/workspace/accounting/PolicyAccountingPage.tsx

```diff
@@ -104,6 +105,7 @@ function PolicyAccountingPage({policy}: PolicyAccountingPageProps) {
     const currentXeroOrganization = findCurrentXeroOrganization(tenants, policy?.connections?.xero?.config?.tenantID);
     const shouldShowSynchronizationError = !!synchronizationError;
     const shouldShowReinstallConnectorMenuItem = shouldShowSynchronizationError && connectedIntegration === CONST.POLICY.CONNECTIONS.NAME.QBD;
+    const shouldShowCardReconciliationOption = cardSettings?.paymentBankAccountID && policy?.areExpensifyCardsEnabled;
```

Comment:
I think it would make more sense to first look if the feature is enabled before looking for the  settings even
```suggestion
    const shouldShowCardReconciliationOption = policy?.areExpensifyCardsEnabled && cardSettings?.paymentBankAccountID;
```